### PR TITLE
Milestone2: Add Reducers and  actions

### PR DESCRIPTION
**Pull request by Favourezeugwa**
## In this milestone, I implemented the following:
- configured Redux Store
- wrote actions and reducer for adding and removing books

### CODE REVIEW

File: src/Redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const CREATE = 'bookstore-app/books/CREATE';
+const REMOVE = 'bookstore-app/books/REMOVE';
+
```
Comment (zieeco) [inline src/Redux/books/books.js]: - Please, kindly remove the added `-app` attached to the `bookstore` to meet the project requirement.
`const CREATE = 'bookstore/books/CREATE'`

Comment (zieeco) [reply src/Redux/books/books.js]: - Please, kindly rename your `Redux` directory to `redux` and `Components` to `components` to meet the structure of the files in the ducks' pattern to meet the project requirement.
- Also remember to update it in your import statement.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Favourezeugwa,


Great work so far! 👏 
You've done well implementing, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Redux/books/books.js (hunk 1 above)
Comment (Favourezeugwa) [reply src/Redux/books/books.js]: sure, will implement them right away

Comment (Favourezeugwa) [reply src/Redux/books/books.js]: I just tried to change the directory names to lower case but it isn't working, although, the subfolders in the Redux directory are all in lower case.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @zieeco, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- You did well to make the changes requested by the previous reviewer 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**